corgidrp/measure_companions.py

```diff
@@ -0,0 +1,516 @@
+import os
+import numpy as np
+import math
+from skimage.transform import resize
+from astropy.table import Table
+import corgidrp.fluxcal as fluxcal
+import pyklip
+import pyklip.fm as fm
+import pyklip.fmlib.fmpsf as fmpsf
+import pyklip.fitpsf as fitpsf
+import pyklip.rdi as rdi
+from corgidrp.data import PyKLIPDataset, Image, Dataset
+import numpy as np
+import corgidrp.fluxcal as fluxcal
+import corgidrp.l4_to_tda as l4_to_tda    
+import astropy.io.fits as fits
+import matplotlib.pyplot as plt
+
+
+def measure_companions(
+    direct_star_image,
+    psf_sub_image,
+    reference_psf,
+    ct_dataset,
+    ct_cal,
+    phot_method='aperture',
+    coronagraphic_dataset=None,
+    fluxcal_factor=None,
+    FpamFsamCal = None,
+    host_star_in_calspec=True,
+    forward_model=False,
+    output_dir = ".",
+    verbose=True
+):
```

Comment:
After discussion with Vanessa, let's assume that host star apparent magnitude is an input to this function as a float. So assume someone else has calculated it for you.